### PR TITLE
i3s: optimization

### PR DESCRIPTION
**Pull request by belom88**
This is improvement for I3S 1.7 loading:
- removed duplicated texture;
- draw indices instead of flattening arrays on CPU;
- optimization for creating MBS from OBB.

It works faster now. But we need to new release after merging this PR to have new version of worker.

### CODE REVIEW

File: modules/i3s/src/lib/parsers/parse-i3s-tile-content.js [hunk 1]
```diff
@@ -373,15 +358,24 @@ function offsetsToCartesians(vertices, metadata = {}, cartographicOrigin) {
  * @returns {object}
  */
 function makePbrMaterial(materialDefinition, texture) {
-  if (!materialDefinition) {
-    return null;
+  let pbrMaterial;
```
Comment (Tamrat-B) [inline modules/i3s/src/lib/parsers/parse-i3s-tile-content.js:361]: +1 on creating default material when not present.

**Review by Tamrat-B — APPROVED** [review]
looks good.

**Review by ibgreen — APPROVED** [review]

**Review by ibgreen — COMMENTED** [review]
Should we add some documentation to the loader explaining that materials will be created etc?

**Comment by belom88** [comment]
> Should we add some documentation to the loader explaining that materials will be created etc?

Sure, I'll do it

**Pull request merged**